src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -57,64 +62,75 @@ class RequestorStep extends React.Component {
             isOnfidoSetupComplete: lodashGet(props, ['achData', 'isOnfidoSetupComplete'], false),
         };
 
+        // Required fields not validated by `validateIdentity`
         this.requiredFields = [
             'firstName',
             'lastName',
-            'requestorAddressStreet',
-            'requestorAddressCity',
-            'requestorAddressZipCode',
-            'dob',
-            'ssnLast4',
-            'requestorAddressState',
+            'isControllingOfficer',
         ];
+
+        // Map a field to the key of the error's translation
+        this.errorTranslationKeys = {
+            firstName: 'bankAccount.error.firstName',
+            lastName: 'bankAccount.error.lastName',
+            isControllingOfficer: 'requestorStep.isControllingOfficerError',
+        };
+
+        this.clearError = inputKey => clearError(this.props, inputKey);
+        this.getErrors = () => getErrors(this.props);
     }
 
-    onFieldChange(field, value) {
+    getErrors() {
```

Comment:
This should be removed since we are defining `this.getErrors` in the constructor?

Reply:
Good catch!